.github/workflows/vast.yml

```diff
@@ -121,15 +121,17 @@ jobs:
           echo "::set-env name=PACKAGE_NAME::$PACKAGE_NAME"
           echo "::set-env name=BUILD_DIR::$BUILD_DIR"
 
+      - name: Export Slug Variables
+        uses: rlespinasse/github-slug-action@v2.x
+
       - name: Fetch ccache Cache
         uses: actions/cache@v2
         with:
           path: .ccache
-          key: ccache-${{ matrix.os.tag }}-${{ matrix.os.compiler }}-${{ matrix.configure.tag }}-${{ github.ref }}-${{ github.sha }}
+          key: ccache-${{ matrix.os.tag }}-${{ matrix.os.compiler }}-${{ matrix.configure.tag }}-${{ env.GITHUB_HEAD_REF_SLUG }}-${{ github.sha }}
```

Comment:
I'm wondering where the cache contents get uploaded, dont we need to apply the same transformation there as well?

Reply:
Yeah valid point. Looks like I'll have to find another way to get the slugs, as the action doesn't run on macOS.

Reply:
Found a better way to get them, and also separated the cache lookups for the different trigger event types.